src/components/ImportExport.vue

```diff
@@ -156,7 +157,10 @@ export default {
     importRoad: function (event) {
       let fail = false;
       // check for legal input
-      if (this.inputtext === '' || this.roadtitle === '') {
+      if (this.inputtext === '' ||
+          this.roadtitle === '' ||
+          this.otherRoadHasName(this.roadtitle)
+      ) {
```

Comment:
I moved your check to here so the user will get some feedback instead of it just failing. Thanks for catching that bug